hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -209,7 +208,7 @@ public static String randomNameOwnedBy(HazelcastInstance instance, String prefix
         for (; ; ) {
             String id = prefix + randomString();
             Partition partition = partitionService.getPartition(id);
-            if (comparePartitionOwnership(false, localMember, partition)) {
+            if (comparePartitionOwnership(true, localMember, partition)) {
```

Comment:
Yes; I also found it confusing. I'll see if I'll clean it up. For the time being it is working